app/experimenter/visualization/api/v3/views.py

```diff
@@ -88,7 +90,30 @@ def computeSignificance(lower, upper):
         return Significance.NEUTRAL
 
 
+def append_conversion_count(results):
+    for branch in results:
+        branch_data = results[branch][BRANCH_DATA]
+        for primary_metric in PRIMARY_METRICS_SET:
+            population_count = branch_data[Metric.USER_COUNT][BranchComparison.ABSOLUTE][
+                "point"
+            ]
+            conversion_percent = branch_data[primary_metric][BranchComparison.ABSOLUTE][
+                "point"
+            ]
+            conversion_count = population_count * conversion_percent
+            branch_data[primary_metric][BranchComparison.ABSOLUTE][
+                "count"
+            ] = conversion_count
+
+
 def process_data_for_consumption(data, experiment):
+    append_population_percentages(data)
+    results = generate_results_object(data, experiment)
+    append_conversion_count(results)
```

Comment:
Mixture of side effectful functions and pure functions here I guess?  Don't really need to change it now since I figure we'll refactor all this later, but just pointing out this could lead to confusing behaviour.

Reply:
Indeed, the backend could use a bit of a refactor. But thanks for pointing this out